Lib/ast.py

```diff
@@ -681,9 +695,15 @@ def visit(self, node):
         self.traverse(node)
         return "".join(self._source)
 
+    def _body_helper(self, node):
```

Comment:
Let's rename this to something like `_rewrite_docstrings_and_traverse_body` (or a better name). I find `body_helper` not very informative.

Reply:
sure! what about `_write_docstring_and_traverse_body`?

Reply:
:ok_hand: